ball_pit/ball_pit.ino

```diff
@@ -448,7 +476,13 @@ void update_display() {
 
 void beep() {
   // The default beep gets slightly higher as the number of balls increases.
-  sound(1000 + (num_balls / 10), 40000);
+  if (is_power_of_two(num_balls)) {
+      play_power_of_two_tune();
+  } else if (is_monodigit(num_balls)) {
+      play_monodigit_tune();
+  } else {
```

Comment:
These comparisons should go in `increment_num_balls` with the other comparisons, not in `beep`.